task4/src/components/Signup.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div class="contents">
+    <h1>新規登録画面</h1>
+    <div class="form">
+      <table>
+        <tr>
+          <td>ユーザ名</td>
+          <td><input type="text" placeholder="UserName" @input="registUserName"></td>
+        </tr>
+        <tr>
+          <td>メールアドレス</td>
+          <td><input type="text" placeholder="E-mail" @input="registEmail"></td>
+        </tr>
+        <tr>
+          <td>パスワード</td>
+          <td><input type="text" placeholder="Password" @input="registPassword"></td>
+        </tr>
+      </table>
+      <button @click="signUp">新規登録</button>
+      <p><a href="#">ログインはこちらから</a></p>
+    </div>
+  </div>
+</template>
+
+<script>
+
+
+export default {
+  name: 'Signup',
+  methods: {
+    registUserName(e) {
+      this.$store.commit('registUserName',e.target.value)

```

Comment:
細かいのですが、,の後ろにはスペースを空けましょう。
フォーマットの自動整形を使用すると良いです。
https://qiita.com/mitashun/items/e2f118a9ca7b96b97840

```suggestion
      this.$store.commit('registUserName', e.target.value)

```